app/src/main/java/com/github/se/orator/model/chatGPT/ChatViewModel.kt

```diff
@@ -22,7 +25,9 @@ import kotlinx.coroutines.launch
 class ChatViewModel(
     private val chatGPTService: ChatGPTService,
     private val apiLinkViewModel: ApiLinkViewModel,
-    private val textToSpeech: TextToSpeech? = null
+    private val textToSpeech: TextToSpeech? = null,
+    private val offlinePromptsFunctions: OfflinePromptsFunctionsInterface =
+        OfflinePromptsFunctions()
```

Comment:
Thank you for adding a default value to keep it compatible with the rest of the code

Reply:
Yeah!